__tests__/e2e/integration/checkNavbarVisibility.ts

```diff
@@ -0,0 +1,56 @@
+describe('When scrolling, the navigation bar must be hidden or shown', () => {
+  before(() => {
+    cy.log(`Visiting http://localhost:3000`)
+    cy.visit('/')
+  })
+
+  it('should show the navigation bar', () => {
+    cy.viewport('iphone-xr')
+    cy.get('#navbar').should(
+      'have.css',
+      'background-color',
+      'rgb(255, 255, 255)'
+    )
+    cy.get('[data-cy=navbar-logo-mobile-container]').should(
+      'have.css',
+      'display',
+      'block'
+    )
+    cy.get('[data-cy=navbar-logo-container]').should(
+      'have.css',
+      'display',
+      'none'
+    )
```

Comment:
no se si realmente es necesario testear que un componente tenga un CSS concreto. Qué opina @DrankoLQ ?

Reply:
Esto lo hago para comprobar que aparece el logo adecuado y que la navbar tiene el color adecuado, que en función del breakpoint se modifica, si no le véis mucho sentido se elimina sin problema @Miangame @DrankoLQ  😄

Reply:
Hmmm me genera dudas lo de testear CSS, no se si es ponerse demasiado el monóculo la verdad 😂 . En todo caso, que se pruebe como un tests de integración e2e si me parece un problema, porque son tests lentos y pensados para otras cosas, por lo que esto debería ir como un test unitario del component, usando `Jest` o `react-testing-library`

Reply:
Deja que revise el código del componente a ver si se me ocurre algo mejor btw

Reply:
Entonces mejor eliminamos tampoco es que aporte un valor muy grande! 👌🏻